travis-ci/vmtest/run.sh

```diff
@@ -385,16 +392,23 @@ if (( SKIPSOURCE )); then
 	echo "Not copying source files..." >&2
 else
 	echo "Copying source files..." >&2
-
 	# Copy the source files in.
 	sudo mkdir -p -m 0755 "$mnt/${PROJECT_NAME}"
-	{
-	if [[ -e .git ]]; then
-		git ls-files -z
+	if [[ "${SOURCE_FULLCOPY}" == "1" ]]; then
+		{
+		if [[ -e .git ]]; then
+			git ls-files -z
+		else
+			tr '\n' '\0' < "${PROJECT_NAME}.egg-info/SOURCES.txt"
```

Comment:
do we even use this .egg-info stuff?.. that might be drgn-specific stuff we inherited

Reply:
i will remove it as i'm also not aware of the use-case for this thing